src/OpenFeature/Model/ProviderEvents.cs

```diff
@@ -36,6 +36,7 @@ public class ProviderEventPayload
         /// <summary>
         /// Metadata information for the event.
         /// </summary>
+        // TODO: This needs to be changed to a EventMetadata object
```

Comment:
I assume we are deferring this as it would be breaking?

If so, we may want to consider how we accumulate and act on such pending breaks?

Reply:
Yes. This would bring breaking changes. As part of the massive breaking changes coming, I created this issue to track this effort: https://github.com/open-feature/dotnet-sdk/issues/234

I've been thinking about creating a milestone to track all the breaking changes so we can safely understand what needs to be done.

Reply:
Something like a milestone seems like a reasonable idea to me.